src/plugins/geomap/_base.scss

```diff
@@ -142,6 +142,7 @@
 
 .olControlAttribution {
 	bottom: 5px;
+	font-size: x-small;
```

Comment:
I don't think these are used anywhere. Would it be possible to convert it to em to be consistent with the rest of the CSS?